### PR TITLE
[Feature] Action to change App group identifier in the Entitlements files

### PR DESCRIPTION
**Pull request by mathiasAichinger**
First version of the action to modify the app group identifier of an app. Just saw that you can specify multiple app groups. I will implement this soon ;)

### CODE REVIEW

File: lib/fastlane/actions/update_app_group_identifier.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+      def self.description
+        "This action changes the app group identifier in the entitlements file"
+      end
+
+      def self.available_options
+        [
+            FastlaneCore::ConfigItem.new(key: :entitlements_file,
+                                         env_name: "FL_UPDATE_APP_GROUP_IDENTIFIER_ENTITLEMENTS_FILE_PATH", # The name of the environment variable
+                                         description: "API Token for UpdateAppGroupIdentifierAction", # a short description of this parameter
```
Comment (KrauseFx) [inline lib/fastlane/actions/update_app_group_identifier.rb]: `API Token for UpdateAppGroupIdentifierAction`?

**Comment by KrauseFx** [comment]
Thank you for the pull request, this looks amazing. Could you also update `docs/Actions.md` with some example uses? :+1:

**Comment by mathiasAichinger** [comment]
I have updated my commit. It now also supports multiple app groups per entitlements file. I also added an entry to the Action.md file.

File: docs/Actions.md [hunk 2]
```diff
@@ -110,6 +110,15 @@ Since you'll probably not want to commit this change into version control, take
 
 **[Show Example Usage](https://github.com/fastlane/examples/blob/4fea7d2f16b095e09af409beb4da8a264be2301e/MindNode/Fastfile#L5-L47)**
 
+### update_app_group_identifiers
+Updates the App Group Identifiers in the given Entitlements file, so you can have  app groups for the app store build and app groups for an enterprise build.
```
Comment (KrauseFx) [inline docs/Actions.md]: Ups, double space after `can have`

**Comment by KrauseFx** [comment]
Looks great, almost ready to merge, sorry for being so picky :wink:

**Comment by mathiasAichinger** [comment]
No problem. ;)

**Comment by KrauseFx** [comment]
Ah, okay, you added everything into one commit. Thanks again @mathiasAichinger for submitting this pull request and contributing to fastlane :+1: :rocket:

**Pull request merged**